paddlespeech/t2s/exps/inference.py

```diff
@@ -120,6 +121,15 @@ def parse_args():
     return args
 
 
+def get_model_suffix(inference_dir, model_name):
+    if os.path.exists(os.path.join(inference_dir, model_name + ".pdmodel")):
```

Comment:
inference support both format support, like:
`paddle.inference.Config(model_name_or_path, model_prefix)`
model_name_or_path: the root path of model
model_prefix: name of model, without suffix

In this time, interface should be fixed:
[[def get_predictor(](https://github.com/PaddlePaddle/PaddleSpeech/blob/67ae7c8dd2317882806c439c038c4cdff3aba896/paddlespeech/t2s/exps/syn_utils.py#L557)](https://github.com/PaddlePaddle/PaddleSpeech/blob/67ae7c8dd2317882806c439c038c4cdff3aba896/paddlespeech/t2s/exps/syn_utils.py#L557)

also for support it compatibility, you can switch it with paddle version.

Reply:
refer https://github.com/PaddlePaddle/PaddleSpeech/pull/3927